Backend/dev/migrations/rider-app/1259-value-add-np.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO atlas_app.value_add_np VALUES (true, 'NAMMA_YATRI', now(), now());
```

Comment:
No need to add this query, I guess in @vsaimanohar PR for migrating tables to dsl.
He removed `white_list_org` table insert queries for local, because of which local had no white listed entries which resulted in no `value_add_np` being present.
I have asked him to create a PR for this fix, will then not require this migration after that.

Reply:
here is the PR link:- https://github.com/nammayatri/nammayatri/pull/6710

Reply:
Merged that, closing it.

Reply:
@jaypalmudaliyar24 @vsaimanohar white_list_org doesn't have the schema the way you're inserting in that table. Can you correct it?

Reply:
@0utkarsh  here is the updated PR: 
https://github.com/nammayatri/nammayatri/pull/6720/commits/09007f34723f29f1a849782e05b85b2827d5c49a